src/helperFunctions/get_config_of_plugins.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+from helperFunctions.fileSystem import get_src_dir
+import configparser
+import logging
+
+
+def get_path_to_conf():
```

Comment:
Prefix function name with a `_`  to show it's not a public method. Also consider merging module with *helperFunctions.config*